Gemfile

```diff
@@ -4,9 +4,9 @@ ruby File.read('.ruby-version').chomp
 
 # Server
 gem 'puma'
-gem 'rails', '~> 7.0'
+gem 'rails', '~> 7.1.2'
 
-gem 'routing-filter', github: 'svenfuchs/routing-filter'
+gem 'routing-filter', github: 'trade-tariff/routing-filter'
```

Comment:
updating `routing-filter` with the one that fixes the 7.1  CSRF compatibility issue.